.landscaper/landscaper-service/resources.yaml

```diff
@@ -0,0 +1,80 @@
+---
+type: blueprint
+name: rbac-blueprint
+relation: local
+input:
+  type: dir
+  path: ./blueprint/rbac
+  mediaType: application/vnd.gardener.landscaper.blueprint.layer.v1.tar+gzip
+  compress: true
+...
+---
+type: blueprint
+name: landscaper-blueprint
+relation: local
+input:
+  type: dir
+  path: ./blueprint/landscaper
+  mediaType: application/vnd.gardener.landscaper.blueprint.layer.v1.tar+gzip
+  compress: true
+...
+---
+type: blueprint
+name: installation-blueprint
+relation: local
+input:
+  type: dir
+  path: ./blueprint/installation
+  mediaType: application/vnd.gardener.landscaper.blueprint.layer.v1.tar+gzip
+  compress: true
+...
+---
+type: landscaper.gardener.cloud/jsonschema
+name: registry-config-definition
+relation: local
+input:
+  type: file
+  path: ./definition/registry-configuration.json
+  mediaType: application/vnd.gardener.landscaper.jsonschema.layer.v1.json
+...
+---
+type: landscaper.gardener.cloud/jsonschema
+name: landscaper-config-definition
+relation: local
+input:
+  type: file
+  path: ./definition/landscaper-configuration.json
+  mediaType: application/vnd.gardener.landscaper.jsonschema.layer.v1.json
+...
+---
+type: ociImage
+name: landscaper-controller-image
```

Comment:
why do we need to duplicate the landscaper images.
We should just reference the original landscaper component descriptor

Reply:
That's correct, it is not needed.
I will use the one from the landscaper component.

Reply:
Just saw that we are currently not adding the images to the landscaper resources:

https://github.com/gardener/landscaper/blob/master/hack/generate-cd.sh#L42

Is this something that needs to be done first?

Reply:
> Is this something that needs to be done first?

this is automatically done by the gardener-ci.
During the component-descriptor creation step in the pipeline there is base component descriptor containing all the images

Reply:
Ok, but I'm not sure how that works.
So where do I find the names under which these resources can be referenced?

Reply:
they are defined in the pipeline definition: https://github.com/gardener/landscaper/blob/master/.ci/pipeline_definitions#L16

Reply:
done